lib/stdlib/test/re_SUITE_data/testoutput1

```diff
@@ -22,6 +22,10 @@ No match
     What do you know about THE QUICK BROWN FOX?
  0: THE QUICK BROWN FOX
 
+/\s/
+    \x20\x20\x20A\x20\x20\x20B\x20\x20\x20C\x20\x20\x20\x20\x20\x20\x20\x20\x20
+ 0: \x20
+
```

Comment:
Place these lines at the end of the file. That will ensure that re_testoutput1_replacement_test.erl will not change.